staging/src/k8s.io/apiserver/pkg/endpoints/handlers/negotiation/negotiate.go

```diff
@@ -119,32 +119,6 @@ func isPrettyPrint(req *http.Request) bool {
 	return false
 }
 
-// negotiate the most appropriate content type given the accept header and a list of
-// alternatives.
-func negotiate(header string, alternatives []string) (goautoneg.Accept, bool) {
```

Comment:
@sttts a while back, we had negotiation problems.  Is this intentionally removed?

Reply:
Looks suspicious. @hanxiaoshuai can you find out when we stopped using this func?

Reply:
func negotiate() was used in func negotiateOutput in v1.4.x , but  func negotiateOutput was remove in v1.5.x. So func negotiate is not used any more. @deads2k @sttts

Reply:
Ok, that's long ago.